enterprise/utils.py

```diff
@@ -2298,6 +2299,22 @@ def batch(iterable, batch_size=1):
         yield iterable[index:min(index + batch_size, iterable_len)]
 
 
+def batch_dict(dict_data, chunk_size=1):
+    """
+    Breaks up a dictionary into equal-sized chunks.
+    No fillers values are added for any 'remainder' chunks
+
+    Arguments:
+        dict (dict): A dictionary to chunk
+        chunk_size (int): the size of each chunk. Defaults to 1.
+    Returns:
+        generator: iterates through each chunk of a dictionary
+    """
+    it = iter(dict_data.items())
+    for _ in range(0, len(dict_data), chunk_size):
+        yield dict(islice(it, chunk_size))
```

Comment:
ah, very nice! you may want to add a couple of unit tests just for this new utility function.